specs-go/v1/layout.go

```diff
@@ -14,8 +14,15 @@
 
 package v1
 
+import "regexp"
+
 // ImageLayout is the structure in the "oci-layout" file, found in the root
 // of an OCI Image-layout directory.
 type ImageLayout struct {
 	Version string `json:"imageLayoutVersion"`
 }
+
+var (
+	// RefsRegexp for string ref should be complied with description "Object names in the refs subdirectories MUST NOT include characters outside of the set of "A" to "Z", "a" to "z", "0" to "9", the hyphen -, the dot ., and the underscore _." at https://github.com/opencontainers/image-spec/blob/master/image-layout.md.
```

Comment:
No need for the URL anymore now that we're in the same repository.  How about:

    // RefsRegexp matches refs that meet the image-layout charset requirements.

or something?